pom.xml

```diff
@@ -43,6 +50,49 @@
         </pluginRepository>
     </pluginRepositories>
 
+    <properties>
+        <workflow.version>1.8</workflow.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>${workflow.version}</version>
+            <scope>test</scope>
```

Comment:
@jglick I thought that the rule would be include `workflow-aggregator`in test scope in order to simplify the dependencies.